src/components/TodoList/TodoList.tsx

```diff
@@ -1,210 +1,103 @@
-            </td>
-
-            <td className="has-text-right is-vcentered">
-              <button data-cy="selectButton" className="button" type="button">
-                <span className="icon">
-                  <i className="far fa-eye-slash" />
-                </span>
-              </button>
-            </td>
-          </tr>
-
-          <tr data-cy="todo">
-            <td className="is-vcentered">4</td>
-            <td className="is-vcentered"><span className="icon" data-cy="iconCompleted"><i className="fas fa-check" /></span></td>
-            <td className="is-vcentered is-expanded">
-              <p className="has-text-success">et porro tempora</p>
-            </td>
+  const todos = useAppSelector(state => state.todos);
+  const currentTodo = useAppSelector(state => state.currentTodo);
+  const query = useAppSelector(state => state.filter.query);
+  const status = useAppSelector(state => state.filter.status);
 
-            <td className="has-text-right is-vcentered">
-              <button data-cy="selectButton" className="button" type="button">
-                <span className="icon">
-                  <i className="far fa-eye" />
-                </span>
-              </button>
-            </td>
-          </tr>
+  const dispatch = useDispatch();
 
-          <tr data-cy="todo">
-            <td className="is-vcentered">5</td>
-            <td className="is-vcentered"> </td>
+  const setCurrentTodo = (todo: Todo) => dispatch(actions.setTodo(todo));
 
-            <td className="is-vcentered is-expanded">
-              <p className="has-text-danger">laboriosam mollitia et enim quasi adipisci quia provident illum</p>
-            </td>
+  const getTodosToDisplay = (todosFromDb: Todo [], curQuery: string, curStatus: string) : Todo [] => {
+    let filteredTodos = [];
 
-            <td className="has-text-right is-vcentered">
-              <button data-cy="selectButton" className="button" type="button">
-                <span className="icon">
-                  <i className="far fa-eye" />
-                </span>
-              </button>
-            </td>
-          </tr>
+    switch (curStatus) {
+      case 'completed':
+        filteredTodos = todosFromDb.filter((todo: Todo) => todo.completed);
+        break;
+      case 'active':
+        filteredTodos = todosFromDb.filter((todo: Todo) => !todo.completed);
+        break;
+      default:
+        filteredTodos = todosFromDb;
+    }
 
-          <tr data-cy="todo">
-            <td className="is-vcentered">6</td>
-            <td className="is-vcentered"> </td>
+    let searchTodos = [];
 
-            <td className="is-vcentered is-expanded">
-              <p className="has-text-danger">qui ullam ratione quibusdam voluptatem quia omnis</p>
-            </td>
+    if (query) {
+      searchTodos = filteredTodos.filter((todo: Todo) => todo.title.includes(curQuery));
+    } else {
+      searchTodos = filteredTodos;
+    }
```

Comment:
You can use a ternary operator

Reply:
Cannot do it without disabling linter which throw an error of expecting assignment instead of expression.
Also don't see reason to use ternary operator here.
So this chunk wasn't fixed

Reply:
Did you try smth like that? 
```suggestion
    searchTodos = query 
      ? filteredTodos.filter((todo: Todo) => todo.title.includes(curQuery))
      : filteredTodos;
```